demos/checkmarx/README.md

```diff
@@ -0,0 +1,38 @@
+# configure checkmarx-plugin
+This is a basic configuration for the [Checkmarx plugin](https://plugins.jenkins.io/checkmarx).
+
+## sample configuration
+Here is a sample configuration using a credential. It is also possible to specify a username and password in the cxscanbuilder section. 
+The filterPattern is the default pattern provided by checkmarx documentation.
+
+```yaml
+credentials:
+  system:
+    domainCredentials:
+      - credentials:
+        - usernamePassword:
+          scope: GLOBAL
+          id: checkmarx_credential
+          description: "login/pwd for checkmarx"
+          username: jenkins_checkmarx
+          password: "StrongPassword"
```

Comment:
use secret interpolation not hardcoding
```suggestion
          password: "${CHECKMARX_PASSWORD}"
```

Reply:
I am not sure to understand your suggestion.
This is the secret definition at system level. It's the only place it appears.

Reply:
OK, this should be provided by other means